CefSharp/IResourceHandler.cs

```diff
@@ -4,18 +4,18 @@
 
 namespace CefSharp
 {
-    public interface IResourceHandler
+    public interface IResourceHandlerFactory
     {
         void RegisterHandler(string url, ResourceHandler handler);
 
         void UnregisterHandler(string url);
 
         /// <summary>
-        /// Called before a resource is loaded. To specify a handler for the resource return a ResourceHandler object
+        /// Called before a resource is loaded. To specify a handler for the resource return a ResourceHandlerFactory object
```

Comment:
Little bit to aggressive with the rename? I think this should still be `ResourceHandler`